schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+              headers:
+                type: object
+                additionalProperties:
+                  $ref: '#Header'
+              content:
+                type: object
+                additionalProperties:
+                  $ref: '#MediaType'
+              links:
+                type: object
+                additionalProperteis:
+                  $ref: '#Link'
+            unevaluatedProperties: false
+      MediaType:
+        $anchor: MediaType
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithSchemaAndExamples'
+        properties:
+          encoding:
+            type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/StyledFormComplexNoDefaultOrEmpty'
+            properties:
+              contentType:
+                type: string
+              headers:
+                type: object
+                additionalProperties:
+                  $ref: '#Header'
+        unevaluatedProperties: false
+      Example:
+        $anchor: Example
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            properties:
+              summary:
+                type: string
+              value: true
+              externalValue:
+                type: string
+                format: uri-reference
+            unevaluatedProperties: false
+      Header:
+        $anchor: Header
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            required:
+              - schema
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/DescReqDep'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - allOf:
+                  - $ref: '#/$defs/Mixins/$defs/WithSchemaAndExamples'
+                  - $ref: '#/$defs/Mixins/$defs/StyledSimple'
+              - allOf:
+                  - $ref: '#/$defs/Mixins/$defs/WithContent'
+                  - $ref: '#/$defs/Mixins/$defs/WithEmpty'
+            unevaluatedProperties: false
+      Paths:
+        $anchor: Paths
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        patternProperties:
+          ^/:
+            type: object
+            oneOf:
+              - $ref: '#Reference'
+              - $ref: '#PathItem'
+        unevaluatedProperties: false
+      PathItem:
+        $anchor: PathItem
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          summary:
+            type: string
+          servers:
+            type: array
+            items:
+              $ref: '#Server'
+          parameters:
+            type: array
+            items:
+              oneOf:
+                - $ref: '#Parameter'
+                - $ref: '#Reference'
```

```suggestion
              $ref: '#Parameter'
```
`Parameter` itself includes `Reference` as a `oneOf`.